src/ui.lua

```diff
@@ -218,8 +218,12 @@ end
 
 function LFGAnnouncementsUI:_createInstanceContainer(instanceId)
 	local instances = Instances
-	local name = instances:GetInstanceName(instanceId)
-
+	local instanceName = instances:GetInstanceName(instanceId)
+	local instanceLevelRange = instances:GetLevelRange(instanceId)
+	local name = instanceName
+	if getn(instanceLevelRange) == 2 then
```

Comment:
level range will always be of size 2. No need to check. (Also, since it's a array it really should be `#instanceLevelRange` for performance)